forc-plugins/forc-client/src/util/account.rs

```diff
@@ -0,0 +1,79 @@
+use async_trait::async_trait;
+use fuel_crypto::{Message, Signature};
+use fuels::{
+    prelude::*,
+    types::{coin_type_id::CoinTypeId, input::Input},
+};
+use fuels_accounts::{wallet::WalletUnlocked, Account};
+
+use super::aws::AwsSigner;
+
+#[derive(Clone, Debug)]
+pub enum ForcClientAccount {
```

Comment:
Can we add a doc comment for this enum?

Reply:
Sure! did that in e9984f1d3311e5012bb304de393def7fe41c4365